### PR TITLE
iOS UI Refresh

### PR DESCRIPTION
**Pull request by stuartbreckenridge**
- [x] Feeds screen

### CODE REVIEW

**Comment by stuartbreckenridge** [comment]
### Feeds

- The Feeds screen now looks more like News, with bold section headers and the disclosure icon moved to the left.
- Folders are bold, with nested items indented to convey the hierarchy. 

nnw-ui

**Comment by gferreroferri** [comment]
Should't folders show the folder icon like the macOS version to keep consistency instead of the > indicator?

**Comment by stuartbreckenridge** [comment]
On the Mac, we show chevron + folder icon + feed. We may go that route here, but my thinking here is that we are space-constrained with fixed sidebar sizes and showing three may be overkill.

**Comment by gferreroferri** [comment]
probably the chevron would not be needed if the folder icon changes between close folder and open folder icon

**Comment by vincode-io** [comment]
@stuartbreckenridge is correct. We don't show the folder icon because of space constraints.

File: iOS/MasterFeed/Cell/MasterFeedTableViewSectionHeader.swift [hunk 1]
```diff
@@ -139,7 +139,6 @@ private extension MasterFeedTableViewSectionHeader {
 	}
 	
 	func commonInit() {
-		addSubviewAtInit(unreadCountView)
```
Comment (vincode-io) [inline iOS/MasterFeed/Cell/MasterFeedTableViewSectionHeader.swift:142]: I'm assuming the rest of the clean up happens in another commit?

Comment (stuartbreckenridge) [reply iOS/MasterFeed/Cell/MasterFeedTableViewSectionHeader.swift:142]: e4cb868

Comment (vincode-io) [reply iOS/MasterFeed/Cell/MasterFeedTableViewSectionHeader.swift:142]: Thanks!

File: iOS/MasterTimeline/MasterTimelineViewController.swift [hunk 2]
```diff
@@ -1052,7 +1052,7 @@ private extension MasterTimelineViewController {
 		let webFeed = coordinator.timelineFeed as! WebFeed
 		let homePageUrlTitle = NSLocalizedString("Copy Home Page URL", comment: "Copy Home Page URL")
 		let homePageCopyURLAction = UIAction(title: homePageUrlTitle,
-											 image: AppAssets.globeImage,
+											 image: AppAssets.safariImage,
```
Comment (vincode-io) [inline iOS/MasterTimeline/MasterTimelineViewController.swift:1055]: Shouldn't these icons match their direct equivalents on the Sidebar context menu?

**Comment by stuartbreckenridge** [comment]
The PR only includes sidebar changes

File: iOS/MasterTimeline/MasterTimelineViewController.swift (hunk 2 above)
Comment (stuartbreckenridge) [reply iOS/MasterTimeline/MasterTimelineViewController.swift]: The do now, but I've removed the context menu work from this PR.

**Comment by gferreroferri** [comment]
Could it be done following Mail.app guidelines? 
0D6F65E7-C783-401B-B3B3-A1CF634CDB65

**Pull request merged**